src/transformers/models/blip/__init__.py

```diff
@@ -0,0 +1,91 @@
+# flake8: noqa
+# There's no way to ignore "F401 '...' imported but unused" warnings in this
+# module, but to preserve other warnings. So, don't check this module at all.
+
+# Copyright 2022 The HuggingFace Team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from typing import TYPE_CHECKING
+
+from ...utils import OptionalDependencyNotAvailable, _LazyModule, is_torch_available, is_vision_available
+
+
+_import_structure = {
+    "configuration_blip": [
+        "BLIP_PRETRAINED_CONFIG_ARCHIVE_MAP",
+        "BlipConfig",
+        "BlipTextConfig",
+        "BlipVisionConfig",
+    ],
+}
+
+try:
+    if not is_vision_available():
+        raise OptionalDependencyNotAvailable()
+except OptionalDependencyNotAvailable:
+    pass
+else:
+    _import_structure["feature_extraction_blip"] = ["BlipFeatureExtractor"]
+    _import_structure["processing_blip"] = ["BlipProcessor"]
+
+
+try:
+    if not is_torch_available():
+        raise OptionalDependencyNotAvailable()
+except OptionalDependencyNotAvailable:
+    pass
+else:
+    _import_structure["modeling_blip"] = [
+        "BLIP_PRETRAINED_MODEL_ARCHIVE_LIST",
+        "BlipModel",
+        "BlipPreTrainedModel",
+        "BlipForConditionalGeneration",
+        "BlipForVisualQuestionAnswering",
+        "BlipVisionModel",
+        "BlipTextModel",
+        "BlipForImageTextRetrieval",
+    ]
+
+if TYPE_CHECKING:
+    from .configuration_blip import BLIP_PRETRAINED_CONFIG_ARCHIVE_MAP, BlipConfig, BlipTextConfig, BlipVisionConfig
+
+    try:
+        if not is_vision_available():
+            raise OptionalDependencyNotAvailable()
+    except OptionalDependencyNotAvailable:
+        pass
+    else:
+        from .feature_extraction_blip import BlipFeatureExtractor
+        from .processing_blip import BlipProcessor
```

Comment:
Same comment here

Reply:
fixed in [bf089c2](https://github.com/huggingface/transformers/pull/20716/commits/bf089c2f05267558025d197aeca9fe4c0457a429)